src/Cli/dotnet/commands/dotnet-test/TestingPlatformCommand.cs

```diff
@@ -96,8 +96,16 @@ public int Run(ParseResult parseResult)
                     VSTestTrace.SafeWriteTrace(() => $"MSBuild task _GetTestsProject didn't execute properly with exit code: {msbuildResult}.");
                     return ExitCodes.GenericFailure;
                 }
+
+                // If not all test projects have IsTestingPlatformApplication set to true, we will simply return
+                if (!_msBuildConnectionHandler.EnqueueTestApplications())
+                {
+                    VSTestTrace.SafeWriteTrace(() => LocalizableStrings.CmdUnsupportedVSTestTestApplicationsDescription);
```

Comment:
This will be replaced by a call to output logger that displays a messages to the UI.

Reply:
Add the todo in the code if we merge before the integration